cpp/include/cuspatial/range/multilinestring_range.cuh

```diff
@@ -154,23 +153,16 @@ class multilinestring_range {
   /// Returns an iterator to the counts of segments per multilinestring
   CUSPATIAL_HOST_DEVICE auto multilinestring_point_count_end();
 
-  /// Returns an iterator to the counts of segments per multilinestring
-  CUSPATIAL_HOST_DEVICE auto multilinestring_segment_count_begin();
-
-  /// Returns an iterator to the counts of points per multilinestring
-  CUSPATIAL_HOST_DEVICE auto multilinestring_segment_count_end();
-
   /// Returns an iterator to the counts of points per multilinestring
   CUSPATIAL_HOST_DEVICE auto multilinestring_linestring_count_begin();
 
   /// Returns an iterator to the counts of points per multilinestring
   CUSPATIAL_HOST_DEVICE auto multilinestring_linestring_count_end();
 
-  /// Returns an iterator to the start of the segment
-  CUSPATIAL_HOST_DEVICE auto segment_begin();
-
-  /// Returns an iterator to the end of the segment
-  CUSPATIAL_HOST_DEVICE auto segment_end();
+  /// @internal
+  /// Returns the owning class that provides views into the segments of the multilinestring range
+  /// Can only be constructed on host
+  auto _segments(rmm::cuda_stream_view);
```

Comment:
Why is this internal?

Reply:
`multilinestring_segment_range` and `multilinestring_segment` are both in detail namespace.